__tests__/core/alerts.test.ts

```diff
@@ -0,0 +1,85 @@
+import { alerts, setAlertsLogLevel } from "../../lib/core";
+
+describe("alerts", () => {
+  beforeEach(() => {
+    console.log = jest.fn();
```

Comment:
I don't think this is _technically_ an issue since my understanding is that each file is isolated by Jest, but could you update this to something along these lines so that `console.log` is always restored after tests:

```typescript
  beforeEach(() => {
    logSpy = jest.spyOn(console, "log").mockImplementation();
  });

  afterEach(() => {
    logSpy.mockRestore();
  });
```

_(feel free to adjust variable names or whatever, mostly going for the idea)_